db/schema.rb

```diff
@@ -606,19 +606,18 @@
     t.index ["payment_id"], name: "index_offline_transaction_charges_on_payment_id"
   end
 
-<<<<<<< HEAD
-  create_table "offline_transaction_refunds", id: :string, force: :cascade do |t|
+  create_table "offline_transaction_disputes", id: :string, force: :cascade do |t|
     t.bigint "payment_id"
     t.datetime "created_at", precision: 6, null: false
     t.datetime "updated_at", precision: 6, null: false
-    t.index ["payment_id"], name: "index_offline_transaction_refunds_on_payment_id"
-=======
-  create_table "offline_transaction_disputes", force: :cascade do |t|
+    t.index ["payment_id"], name: "index_offline_transaction_disputes_on_payment_id"
+  end
+
+  create_table "offline_transaction_refunds", id: :string, force: :cascade do |t|
     t.bigint "payment_id"
     t.datetime "created_at", precision: 6, null: false
     t.datetime "updated_at", precision: 6, null: false
-    t.index ["payment_id"], name: "index_offline_transaction_disputes_on_payment_id"
->>>>>>> 29887023... Initial offline_transaction_dispute support
```

Comment:
Some rebase was wrong here, I think it would be better to fix it on your branch!

Reply:
Dang it, that's on me. I just force pushed.